lib/lib.ml

```diff
@@ -40,9 +40,8 @@ let validate_protocols_exn (ast : scr_module) ~verbose : unit =
   let g_types =
     List.map ~f:(fun p -> (Gtype.of_protocol p, p.value.roles)) protocols
   in
-  let g_types =
-    List.map ~f:(fun (g, roles) -> (Gtype.normalise g, roles)) g_types
-  in
+  (* let g_types = List.map ~f:(fun (g, roles) -> (Gtype.normalise g, roles))
```

Comment:
if don't normalise the g then we could remove it. Or was it temporary?

Reply:
https://github.com/nuscr/nuscr/issues/130